MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# .NET MAUI Image Editor (SfImageEditor) Image Zooming
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
```

Comment:
The image editor control in the .NET MAUI provides support for zooming and panning actions on an image.